app/js/controllers/gesture.js

```diff
@@ -0,0 +1,85 @@
+/*global Controller*/
+
+const ACTIVE_WINDOW_SELECTOR = '#windows > .active iframe';
+
+export default class GestureController extends Controller {
+  constructor(options) {
+    super(options);
+
+    var firstTouchStartEvt = null;
+    var isForwarding = false;
+
+    window.addEventListener('touchstart', (evt) => {
+      if (evt.touches.length === 1) {
+        firstTouchStartEvt = evt;
+        return;
+      }
+
+      if (evt.touches.length !== 2) {
+        return;
+      }
+
+      var iframe = document.querySelector(ACTIVE_WINDOW_SELECTOR);
+      iframe.sendTouchEvent.apply(iframe, unsynthesizeEvent(firstTouchStartEvt));
+      iframe.sendTouchEvent.apply(iframe, unsynthesizeEvent(evt));
+
+      isForwarding = true;
+    });
+
+    window.addEventListener('touchmove', (evt) => {
+      if (!isForwarding) {
+        return;
+      }
+
+      var iframe = document.querySelector(ACTIVE_WINDOW_SELECTOR);
+      iframe.sendTouchEvent.apply(iframe, unsynthesizeEvent(evt));
+    });
+
+    window.addEventListener('touchend', (evt) => {
+      if (!isForwarding) {
+        return;
+      }
+
+      if (evt.touches.length === 0) {
+        isForwarding = false;
+      }
+
+      var iframe = document.querySelector(ACTIVE_WINDOW_SELECTOR);
+      iframe.sendTouchEvent.apply(iframe, unsynthesizeEvent(evt));
+    });
+
+    console.log('[Customizer] Initialized GestureController', this);
+  }
+}
+
+/**
+ * Taken from System app:
+ * https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/touch_forwarder.js#L93
```

Comment:
We should link to a specific hash, as this may move around with more work on Gaia.